h5py/_hl/files.py

```diff
@@ -293,7 +296,7 @@ def swmr_mode(self, value):
     def __init__(self, name, mode=None, driver=None,
                  libver=None, userblock_size=None, swmr=False,
                  rdcc_nslots=None, rdcc_nbytes=None, rdcc_w0=None,
-                 track_order=None,
+                 track_order=None, strategy=None, persist=False, threshold=1,
```

Comment:
I think all three of these names should have a prefix to make it clear what they relate to, e.g. `file_space_strategy`.